### DISCUSSION TITLE
Are "starters" licensed the same way as react-spectrum?

### DISCUSSION THREAD

**Discussion by raix** [adobe/react-spectrum#5661] [Q&A]
Does the [LICENSE](https://github.com/adobe/react-spectrum/blob/main/LICENSE) align with the intent described in the [starters README](https://github.com/adobe/react-spectrum/tree/tw-starter/starters#readme)?
Reference:
<img src="https://github.com/adobe/react-spectrum/assets/1136718/9f72246f-5648-41ab-8e3e-636ced82dba9" height="150" />

I'm asking as we would love to use the new RAC tailwind starters in [PlatformPlatform](https://github.com/platformplatform/PlatformPlatform) as a solid starting point for developers who care about accessibility.

What's the guidance / expectations in terms of license and credit etc. for usage of starters in downstream open source projects?

Maybe @devongovett knows?

Kind regards Morten

**Comment by snowystinger** [comment]
Right now, the License for the project applies to everything, including the starter. https://github.com/adobe/react-spectrum/blob/main/LICENSE

We are discussing some alternatives to hopefully make that a little easier.

**Reply by raix** [reply]
Thanks @snowystinger sounds great - the starter kits feel more like starter templates, maybe a different repo or a RAC create tool - generating code for the user to "own". _(similar approach to Shadcn)_

We will hold and wait until further notice - let me know if we can help out in anyway on the starter project _(we had planned similar work https://github.com/platformplatform/PlatformPlatform/issues/210)_

**Reply by devongovett** [reply]
Following up on this: the answer is to include the `LICENSE` file in your repo along with the source files, e.g. in the same folder. That's now included in the downloadable ZIP files as well.
